_data/changelogs/component-tooltip.yml

```
@@ -5,6 +5,14 @@ type: component
 changelogURL:
 
 items:
+  - date: NNNN-NN-NN
+    summary: Updated component behavior to allow users to hover over tooltip content.
+    summaryAdditional: This allows the component to meet the "hoverable" standard outlined in [WCAG 1.4.13](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html).
```

While I generally prefer the display of content in this link, I feel like we should update this link to match the one used in uswds/uswds-site#2646 for consistency. Let me know what you think!

```suggestion
    summaryAdditional: This allows the component to meet the "hoverable" standard outlined in [WCAG 1.4.13](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus).
```

---

I dig it. Thanks @mahoneycm. This was merged in [4770585](https://github.com/uswds/uswds-site/pull/2661/commits/47705851b17309e6de5d32661d9b7a0565e88199).